test/catch.hpp

```diff
@@ -10545,7 +10545,7 @@ namespace Catch {
             // Extracts the actual name part of an enum instance
             // In other words, it returns the Blue part of Bikeshed::Colour::Blue
             StringRef extractInstanceName(StringRef enumInstance) {
-                // Find last occurence of ":"
+                // Find last occurrence of ":"
```

Comment:
This file is generated by https://github.com/catchorg/Catch2 - please make the contribution there and then I'll be happy to ingest a new version from that project.

Reply:
Pull request is out: https://github.com/catchorg/Catch2/pull/2206
I don't think that it's worth ingesting a new version just for such a small cosmetic change and I definitely won't be offended if you wait for a more valuable payload before making a new ingestion :)